### PR TITLE
add sdk section and mention permissionless there

### PR DESCRIPTION
**Pull request by nikmel2803**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -29,6 +30,9 @@ If necessary, the contract owner can withdraw tokens using the withdrawToken fun
 To update the token price, call the updatePrice function.
 For more information, please refer to the comments within the contract source code.
 
+## SDK
+You can use permissionless.js to interact with an ERC20 Paymaster. Follow [this link](https://docs.pimlico.io/permissionless/tutorial/tutorial-3) to check out tutorial on how to submit user operation using with an ERC-20 Paymaster.
```
[the remark this answers is not in the document]
Comment (kristofgazso) [reply README.md:34]: otherwise good to merge!

File: README.md [hunk 2]
```diff
@@ -29,6 +30,11 @@ If necessary, the contract owner can withdraw tokens using the withdrawToken fun
...
+
+Check out [this docs](https://docs.pimlico.io/permissionless/how-to/paymasters/use-custom-paymaster) if you're going to use custom paymaster implementation.
```
Comment (kristofgazso) [inline README.md:36]: grammar

**Review by kristofgazso — CHANGES_REQUESTED** [review]

File: README.md [hunk 3]
```diff
@@ -29,6 +30,11 @@ If necessary, the contract owner can withdraw tokens using the withdrawToken fun
 To update the token price, call the updatePrice function.
 For more information, please refer to the comments within the contract source code.
 
+## SDK
+You can use permissionless.js to interact with an ERC20 Paymaster. Follow [this link](https://docs.pimlico.io/permissionless/tutorial/tutorial-3) to check out tutorial on how to submit user operation using with an ERC-20 Paymaster.
+
+Check out [these docs](https://docs.pimlico.io/permissionless/how-to/paymasters/use-custom-paymaster) if you're going to use custom paymaster implementation.
```
Comment (kristofgazso) [inline README.md:36]: ...use _a_ custom paymaster...

**Review by nikmel2803 — COMMENTED** [review]
article fix

**Pull request merged**